tools-local/tasks/mobile.tasks/WasmBuildTasks/CompileTimeZoneData.cs

```diff
@@ -81,17 +84,21 @@ public override bool Execute()
             Log.LogError($"Could not find required file {ZoneTabFile}");
             return false;
         }
-
         CompileTimeZoneDataSource();
 
-        string[] filtered = new string[] { "America/Los_Angeles", "Australia/Sydney", "Europe/London", "Pacific/Tongatapu",
+        if (FilterSystemTimeZones!) {
```

Comment:
Why is the `!` needed here? Also, opening brace should be on a new line.

Reply:
The `!` is needed when you have a nullable type, like `public string[]? TimeZones`, and you want to tell the compiler that you know it isn't null, so it shouldn't complain about a possible null dereference. And because `FilterSystemTimeZones` is not nullable, `!` isn't required here. (https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/null-forgiving)

Reply:
Oops it was original nullable but I forgot to take the exclamation point out.